jekyll/_cci2/overview.md

```diff
@@ -10,7 +10,7 @@ This article provides System Administrators with an overview of CircleCI 2.0 ins
 * TOC
 {:toc}
 
-CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. CircleCI is installable inside your private cloud or data center and is free to try for a limited time. 
+CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. CircleCI is installable inside your private cloud or data center and is free to try for a limited time. Contact us at [Support](https://support.circleci.com/hc/en-us) to request a trial license.
```

Comment:
is this really what we are supposed to tell people ? do we no longer use this ? https://circleci.com/enterprise-trial-install i send it to customers all the time cuz it's much easier than having them ask us to create & send a license

Reply:
I just copied what was already there, I am not sure which is preferred (this is the new link to the support center), so I can change it if you think the web form is better.

Reply:
mmmm yeah i didn't realize it was already there.... either one seems fine then lol 🤷‍♀️

Reply:
I changed it to the web form, thanks!